architecture/core_concepts/routes.adoc

```diff
@@ -420,36 +420,95 @@ default certificate] will be used for TLS termination.
 apiVersion: v1
 kind: Route
 metadata:
-  name: route-edge-secured
+  name: route-edge-secured <1>
 spec:
   host: www.example.com
   to:
     kind: Service
-    name: service-name
+    name: service-name <1>
   tls:
-    termination: edge            <1>
-    key: |-                      <2>
+    termination: edge            <2>
+    key: |-                      <3>
       -----BEGIN PRIVATE KEY-----
       [...]
       -----END PRIVATE KEY-----
-    certificate: |-              <3>
+    certificate: |-              <4>
       -----BEGIN CERTIFICATE-----
       [...]
       -----END CERTIFICATE-----
-    caCertificate: |-            <4>
+    caCertificate: |-            <5>
       -----BEGIN CERTIFICATE-----
       [...]
       -----END CERTIFICATE-----
 ----
-<1> The `*termination*` field is `edge` for edge termination.
-<2> The `*certificate*` field is the contents of the PEM format certificate file.
-<3> The `*key*` field is the contents of the PEM format key file.
-<4> An optional CA certificate may be required to establish a certificate chain for validation.
+<1> The name of the object, which is limited to 63 characters.
+<2> The `*termination*` field is `edge` for edge termination.
+<3> The `*certificate*` field is the contents of the PEM format certificate file.
+<4> The `*key*` field is the contents of the PEM format key file.
+<5> An optional CA certificate may be required to establish a certificate chain for validation.
 ====
 
 Because TLS is terminated at the router, connections from the router to
 the endpoints over the internal network are not encrypted.
 
+Edge-terminated routes can specify an `insecureEdgeTerminationPolicy` that
```

Comment:
@bfallonf These lines (454 - 511) are for a 3.1-only feature (see https://github.com/openshift/openshift-docs/pull/1140). It's getting pulled in here from one of the commits. We'll need to remove this before merging to 3.0 docs.